internal/corazawaf/rule.go

```diff
@@ -174,10 +175,12 @@ func (r *Rule) doEvaluate(phase types.RulePhase, tx *Transaction, cache map[tran
 		rid = r.ParentID_
 	}
 
+	logger := tx.DebugLogger().With(debuglogger.Int("rule_id", rid))
```

Comment:
Creating logger instance here is resulting in additional heap allocations and is impacting performance.  Can we just update the context with new "rule_id" instead of creating new logger instance

Reply:
Rolled back.